internal/fnruntime/runner.go

```diff
@@ -246,7 +246,7 @@ func parseNameAndNamespace(yml *yaml.RNode, fnResult *fnresult.Result) error {
 	return nil
 }
 
-func populateResourceRef(item *yaml.RNode, resultItem *framework.ResultItem) error {
+func populateResourceRef(item *yaml.RNode, resultItem *framework.Result) error {
```

Comment:
unrelated, but IIRC this code was always meant to be temporary to support some backwards compatibility. Any plans to remove it?